packages/components/src/templates/next/components/complex/CollectionBlock/utils/getCollectionParent.ts

```diff
@@ -0,0 +1,29 @@
+import type { IsomerSiteProps } from "~/types"
+import type { IsomerCollectionPageSitemap } from "~/types/sitemap"
+import { getSitemapAsArray } from "~/utils/getSitemapAsArray"
+
+interface GetCollectionParentProps {
+  site: IsomerSiteProps
+  collectionId: string
+}
+
+export const getCollectionParent = ({
+  site,
+  collectionId,
+}: GetCollectionParentProps): IsomerCollectionPageSitemap => {
+  const items = site.siteMap.children?.flatMap((child) =>
+    getSitemapAsArray(child),
+  )
```

Comment:
minor nit: I think we should be able to do `getSitemapAsArray(site.siteMap)` directly